cloud/passportauth/README.md

```diff
@@ -0,0 +1,75 @@
+# RainCatcher Passport Auth
+Raincatcher authentication and authorization module using [PassportJS][1]. This module creates and initializes a Passport authentication service using Passport's local strategy.
+This allows express routes to be protected from requests by checking if the user is authenticated and/or authorized with the required role. This module also
```

Comment:
"This module also"

If there is more than 3 responsibilities (and I think there are) list list them using bullet points
Use common facts and avoid long sentences.